operator/Makefile

```diff
@@ -449,8 +449,7 @@ bundle-post-process: test-bundle-helpers operator-sdk ## Post-process CSV file t
 	mkdir -p build/
 	rm -rf build/bundle
 	cp -a bundle build/
```

Comment:
Please move these lines (and the comment above which GH does not let me select) into the script as well.
Then rename the script to something like `prepare-bundle-manifests.sh`